ddev/src/ddev/cli/release/changelog.py

```diff
@@ -0,0 +1,125 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import click
+
+if TYPE_CHECKING:
+    from ddev.cli.application import Application
+
+
+@click.command(short_help='Add or fix changelog entries')
+@click.argument('entry_type', required=False)
```

Comment:
As a reminder, we can do both validation and prompting with click:

- https://click.palletsprojects.com/en/8.1.x/options/#option-prompting
- https://click.palletsprojects.com/en/8.1.x/options/#choice-options